### ISSUE TITLE
Exception in SetHighlightTexture 

### ISSUE THREAD

**Issue by CrenshawCoding** [Spielstein42/GroupfinderFlags#1]
Exception occurs when entering world:

```
166x bad argument #3 to '?' (Usage: self:SetHighlightTexture(asset [, blendMode]))
[string "=[C]"]: in function `SetHighlightTexture'
[string "@GroupfinderFlags/main.lua"]:123: in main chunk

Locals:
(*temporary) = "bad argument #3 to '?' (Usage: self:SetHighlightTexture(asset [, blendMode]))"
```

**Comment by Spielstein42** [comment]
That should be fixed in the addon version 0.15a. I pushed the corresponding code to this repo. The repo lacked behind as I recently only uploaded the current/updated versions directly to CurseForge.

**Comment by CrenshawCoding** [comment]
Okay I’ll update 👍 think my cursebreaker downloads from wowinterface.
